workers/unity/Assets/Fps/Scripts/SetupLogic/WorkerConnectorBase.cs

```diff
@@ -11,7 +12,7 @@ public abstract class WorkerConnectorBase : DefaultWorkerConnector
 
         [SerializeField] protected MapBuilderSettings MapBuilderSettings;
 
-        protected GameObject levelInstance;
+        public GameObject LevelInstance;
```

Comment:
I'm not clear on what the intended usage is of the public LevelInstance reference. Is it supposed to be hidden in the inspector? If not, what object is to be referenced by it?

Reply:
Its used by the map builder - so has to be public (or internal I guess)

Reply:
Does it need to be accessed from the inspector? If not, a `[HideInInspector]` attribute would be preferable to avoid people thinking it's something they need to worry about, or if it doesn't need to be serialized, `[NonSerialized]`

Reply:
I don't think its accessed from the inspector - will add the attribute 👍